src/Engine/Scene/CameraComponent.hpp

```diff
@@ -13,6 +13,7 @@ namespace Scene {
 class Entity;
 
 /// A Camera is an Engine Component storing a Camera object.
+///\todo attached camera do not take entity's transformation into account ...
```

Comment:
Do this means that if the Entity is animated (e.g. keyframed) there is no impact on the Camera ?

Reply:
yes, but if the camera is animated, the component's RO is updated.